contracts/upgradeable_contracts/amb_native_to_erc20/ForeignAMBNativeToErc20.sol

```diff
@@ -147,7 +146,7 @@ contract ForeignAMBNativeToErc20 is BasicAMBNativeToErc20, ReentrancyGuard, Base
     * @param _data this parameter could contain the address of an alternative receiver of the tokens on the other network,
     * otherwise it will be empty.
     */
-    function onTokenTransfer(address _from, uint256 _value, bytes _data) external bridgeMessageAllowed returns (bool) {
```

Comment:
Consider to increase the minor in the version here: https://github.com/poanetwork/tokenbridge-contracts/blob/f58cff263c4b58354fef253a3c1827cc4c06658d/contracts/upgradeable_contracts/amb_native_to_erc20/BasicAMBNativeToErc20.sol#L64-L66